tests/Responder/FormattedResponderTest.php

```diff
@@ -86,7 +81,7 @@ public function testSorting()
 
     /**
      * @expectedException InvalidArgumentException
-     * @expectedExceptionMessageRegExp /formatters .* must implement .*AbstractFormatter/i
+     * @expectedExceptionMessageRegExp /formatters .* must implement .*FormatterInterface/i
```

Comment:
Do we have a standard for setting expected exceptions in annotations versus method calls?